mllib/src/test/scala/org/apache/spark/ml/util/MLTest.scala

```diff
@@ -109,4 +110,14 @@ trait MLTest extends StreamTest with TempDirectory { self: Suite =>
     testTransformerOnDF(dataframe, transformer, firstResultCol,
       otherResultCols: _*)(globalCheckFunction)
   }
+
+  def testPredictorModelSinglePrediction(model: PredictionModel[Vector, _],
```

Comment:
Thanks for the updates!  Just this comment: Can we please call this method testPredictionModelSinglePrediction to match the name of the class it is testing?